### ISSUE TITLE
Move the DELETE call from the add-on to the integration

### ISSUE THREAD

**Issue by IngmarStein** [savdagod/TidbytAssistant#8]
The integration already calls the Tidbyt API directly to set the brightness. It could to the same to delete app installations and save a roundtrip through the add-on.

**Comment by savdagod** [comment]
Good call, I’ll work on doing that then we can remove it from the addon (or leave it incase someone wants to use the addon outside of HomeAssistant)

**Comment by savdagod** [comment]
Added to v1.0.9

**Comment by motoridersd** [comment]
It seems the direct API Delete is not working on a Gen2, but it works fine on a Gen1. If I do a pixlet delete or use the app, I can remove it.

Is there any logging I can enable to debug this? I'm seeing no errors in the add-on, but seeing this issue I now know why, since it's not the add on that is trying to delete the app.

**Comment by motoridersd** [comment]
There's no error or any activity logged, but the app remains on the 2nd Gen Tidbyt. I have to manually remove it with Pixlet or through the app.

I tried using the Dev Tools to do the service call directly and that worked... I kept running the action from the Automation and it wasn't working, nor was it getting removed when the automation ran.

I deleted the action from the automation and added it again and that seems to have done it... It was odd that I was getting no errors and the app wasn't being removed.
